guide/07-working-with-bigdata/03-performing_big_data_analysis.ipynb

```diff
@@ -0,0 +1,956 @@
+{
```

Comment:
<p>The aggregate points tool returns a feature layer item that contains the processed results.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>